src/components/router/Router.js

```diff
@@ -0,0 +1,42 @@
+import { renderMainPage } from '../../views/MainPage/MainPage.js';
+import { renderQuizSettings } from '../../views/quiz-settings/quiz-settings.js';
+import { renderQuizView } from '../../views/QuizView/quizView.js';
+import { renderScorePage } from '../../views/ScorePage/ScorePage.js';
+import { renderLeaderboard } from '../../views/Leaderboard/Leaderboard.js';
+import { renderAdoptionPage } from '../../views/AdoptionPage/AdoptionPage.js';
+import { renderCreditsPage } from '../../views/CreditsPage/CreditsPage.js';
+
+const app = document.getElementById('app');
+
+const applicationRoutes = {
+  '/': renderMainPage,
+  '#quiz-settings': renderQuizSettings,
+  '#quiz': renderQuizView,
+  '#score-page': renderScorePage,
+  '#leadearboard': renderLeaderboard,
+  '#adoption-page': renderAdoptionPage,
+  '#credits-page': renderCreditsPage,
+};
+
+function redirectToMainPage() {
+  window.location.hash = '';
+  return applicationRoutes['/'];
+}
+
+function setRenderFunction(path) {
+  if (applicationRoutes[path] === undefined) {
+    return redirectToMainPage;
+  } else {
+    return applicationRoutes[path];
+  }
```

Comment:
```suggestion
  }
  return applicationRoutes[path];
```
else is not necessary

Reply:
ok